packages/manager/modules/manager-components/src/index.js

```diff
@@ -0,0 +1,9 @@
+import angular from 'angular';
+
+import resourceSelector from './resource-selector';
+
+const moduleName = 'ovhManagerManagerComponents';
+
+angular.module(moduleName, [resourceSelector]);
+
+export default moduleName;
```

Comment:
This will export all components in this module irrespective of them being used or not. Lets find a way to support only components needed like the way date-fn does.

Reply:
Yeah... Probably, we can have it as a separate story...